src/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.photo-Mac {
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
```

Comment:
Please add around paddings for card instead adding around margin, padding for each element. It should be smth like this:
![image](https://user-images.githubusercontent.com/54936044/228689483-4e113870-0e40-49ea-9524-fcdb1969fcd9.png)